platform-sdk/swirlds-unit-tests/common/swirlds-test-framework/src/main/java/com/swirlds/test/framework/config/TestConfigBuilder.java

```diff
@@ -74,7 +78,8 @@ public TestConfigBuilder(@Nullable final Class<? extends Record> dataTypes) {
      */
     public TestConfigBuilder(final boolean registerAllTypes) {
         if (registerAllTypes) {
-            this.builder = ConfigUtils.scanAndRegisterAllConfigTypes(ConfigurationBuilder.create());
+            this.builder = ConfigUtils.scanAndRegisterAllConfigTypes(
+                    ConfigurationBuilder.create(), Set.of(SWIRLDS_PACKAGE, HEDERA_PACKAGE));
```

Comment:
You could define the Set directly as a constant without introducing `SWIRLDS_PACKAGE` and `HEDERA_PACKAGE`